compiler/src/main/scala/org/scalajs/nscplugin/CompatComponent.scala

```diff
@@ -64,6 +64,8 @@ trait CompatComponent {
   def isImplClass(sym: Symbol): Boolean =
     scalaUsesImplClasses && sym.hasFlag(Flags.IMPLCLASS)
 
+  lazy val isScala211: Boolean = scalaUsesImplClasses
```

Comment:
Makes me wonder if we should use a more explicit test for `2.11` here :-/ But maybe this is OK.